### ISSUE TITLE
<video> ?

### ISSUE THREAD

**Issue by beaufortfrancois** [GoogleChromeLabs/sample-pie-shop#21]
Do you have any plan to add videos to product page?
If so, I'd be happy to help there and use shaka player to provide a delighful media (offline) experience.

**Comment by samdutton** [comment]
Good idea — let's look at this once we have the basic site up and running.
